### ISSUE TITLE
CTGAN algorithm does not respect the constraints for minimum and maximum bound imposed on a column Tabular dataset..

### ISSUE THREAD

**Issue by krenusz** [sdv-dev/SDV#366]
### Problem Description

In our solution we needed two greaterthan constraints. Since the greaterthan constraint does not accept integer values for the lower bound, we created two dummy columns in the dataset to reference in the constraint. However the constraints are not respected in the synthetized data.

### Additional context
```python
def synthesis(data,number_of_samples,key,name):
    from sdv.tabular import CTGAN
    from sdv.constraints import GreaterThan
    

    pin_constrait_low = GreaterThan(low='relative low',high='n_pin',handling_strategy='transform')
    pin_constrait_high = GreaterThan(low='n_pin',high='relative high',handling_strategy='transform')    
    constraints = [pin_constrait_high,
                   pin_constrait_low]
    
        
    '''CTGAN Deeplearning approach'''    
    model = CTGAN(epochs=50, primary_key=key, constraints=constraints, field_transformers={'Result':'categorical_fuzzy'})

    model.fit(data)
    new_data = model.sample(number_of_samples)

      
    return new_data
```
### Ability to contribute
I think it would be an optimal solution to allow integer values to be set as a bound and beyond that I would allow equal relation as an option. An other suggestion to allow column list as an input activate the constraints on more than one columns.

**Comment by pvk-developer** [comment]
Hi @krenusz , thank you for opening this issue, here you are mentioning a couple of functionalities that we don't have, to which I have opened the following feature requests to be implemented in the future: #367 and #368

**Comment by krenusz** [comment]
@pvk-developer  Perfect, Thank You!
